docs/security/fips_support.md

```diff
@@ -6,15 +6,15 @@ FIPS 140-2 is a U.S. Federal Government security standard used to approve crypto
 
 ## Use of FIPS Compatible Go compiler.
 
-The Go compiler in use can be found [here](https://hub.docker.com/u/goboring). Each component of the system is built with the version of this compiler that matches the same standard Go compiler version that would be used otherwise.
+The Go compiler in use can be found [here](https://go.googlesource.com/go/+/dev.boringcrypto). Each component of the system is built with the version of this compiler that matches the same standard Go compiler version that would be used otherwise.
```

Comment:
You might link to the branch readme: https://github.com/golang/go/blob/dev.boringcrypto/README.boringcrypto.md